### ISSUE TITLE
"This repository is empty."

### ISSUE THREAD

**Issue by almereyda** [NixOS/nixops-dashboard#1]
As an interested individual, I can not imagine what, why, who, when and where rescourses about this project can be found. Is the a blog post or something?

**Comment by domenkozar** [comment]
It's coming any time now :)

**Comment by spinus** [comment]
is "any time now" more "now" now?
Maybe would be enough to put a draft to the repo or a readme and interested people (like me) could help?

**Comment by fstamour** [comment]
Is @domenkozar trolling? Saying such things at such date :cry:

**Comment by spinus** [comment]
ah, got me :-)

**Comment by RostakaGmfun** [comment]
So, are there any updates?

**Comment by davidak** [comment]
@RostakaGmfun no

**Comment by davidak** [comment]
@domenkozar does anything more than the idea described in the name exists?

Will there ever be content in this repo?

**Comment by domenkozar** [comment]
It's not up to me unfortunately, but I'll check the status.

**Comment by rbvermaa** [comment]
@davidak Yeah, something exists, but I am not sure if Infor/LogicBlox is planning to release it. I have been gone over a year now. Thinking of removing the repo for now.

**Comment by davidak** [comment]
@rbvermaa thanks. Maybe someone still working there could ask?

**Comment by Thesola10** [comment]
Maybe a web-based dashboard, a bit like [Portainer](https://github.com/portainer/portainer) or the Project Atomic one, could really go to show the true potential of NixOS as a server platform. This repo could serve as a reminder that this exists as a goal?

**Comment by kaliumxyz** [comment]
Is this up for grabs for anyone or group willing to work on it?

**Comment by hugolgst** [comment]
> It's coming any time now :)

Some time has elapsed since then.
